src/content/reference/react-dom/server/renderToStaticMarkup.md

```diff
@@ -16,62 +16,62 @@ const html = renderToStaticMarkup(reactNode)
 
 ---
 
-## Reference {/*reference*/}
+## Referensi {/*reference*/}
 
 ### `renderToStaticMarkup(reactNode)` {/*rendertostaticmarkup*/}
 
-On the server, call `renderToStaticMarkup` to render your app to HTML.
+Pada *server*, panggil fungsi `renderToStaticMarkup` untuk me-*render* aplikasi Anda ke HTML.
 
 ```js
 import { renderToStaticMarkup } from 'react-dom/server';
 
 const html = renderToStaticMarkup(<Page />);
 ```
 
-It will produce non-interactive HTML output of your React components.
+Keluaran yang diberikan adalah HTML non-interaktif dari komponen-komponen React Anda.
 
-[See more examples below.](#usage)
+[Lihat lebih banyak contoh dibawah.](#usage)
 
-#### Parameters {/*parameters*/}
+#### Parameter {/*parameters*/}
 
-* `reactNode`: A React node you want to render to HTML. For example, a JSX node like `<Page />`.
+* `reactNode`: Sebuah *node* React yang Anda ingin *render* ke HTML. Contohnya, sebuah *node* JSX seperti `<Page />`.
 
-#### Returns {/*returns*/}
+#### Nilai Balik {/*returns*/}
 
-An HTML string.
+Sebuah *string* HTML.
 
-#### Caveats {/*caveats*/}
+#### Anjuran {/*caveats*/}
```

Comment:
Keep it as `Caveats`

Reply:
In `startTransition` I also changed to `Caveats` to `Anjuran`, so I synced them both by changing it back to `Caveats` as well in [c407600](https://github.com/reactjs/id.react.dev/pull/505/commits/c407600525272d31d8b8d6bf9fc9630609030ce2) 👍

Reply:
@mazipan @r17x @resir014 @zainfathoni Since there are other PRs already using `Caveat` as it is, then later on we'll have to revise them.